CIP-XXXX/README.MD

```diff
@@ -0,0 +1,146 @@
+
+- **Key Registration:** Once derived, register these stake keys. This registration process remains consistent with current procedures, with the added context that all these new stake keys will be associated with a common payment key.
+
+- **Fund Distribution:** Distribute the funds across addresses formed by pairing the first  payment key with each of the newly created stake keys. This creates an association between all stake delegations. The details of how funds are distributed are specified by the user when creating the multi delegation transaction.
+
+Wallet software can then easily discover all stake keys and associated delegations either by sequential discovery or identifying addresses  containing the first payment credential. 
+
+### Updating the Multi-Delegation
+The dynamic nature of multi-delegation facilitates not just participation but also continuous optimization of delegations.
+
+#### Adding Pools
+To delegate to additional pools, the first step would involve deriving the necessary number of new stake keys. Then, make a transaction that:  registers the new stake keys, delegates and  distributes funds among addresses that pair the new stake keys with the first payment key.
+
+#### Removing Pools
+To remove a pool, the wallet first identifies all addresses associated with the stake key designated for removal. Then, make a transaction that
+Transfer all funds from these addresses
+withdraw all rewards from their associated reward accounts to one of the accounts that are still in delegation. If no other delegating accounts are left, revert the funds to addresses associated with the first payment key. 
+. Deregister stake keys
+
+**Note that all rewards calculated but not distributed will be lost.**
+
+#### Interoperability
+Live delegation portfolio might drift overtime due to:
+- Receiving funds to a single address 
+- Building transactions with software that doesn’t support multi-stake delegation (e.g. via dapp connector)
+
+In order to maintain a record of user’s delegation portfolio preferences , all actions pertaining to stake delegation, including funds transfers, stake key registration, and deregistration, must be executed within a single atomic transaction when performing an update. This transaction should also include the delegation portfolio metadata, using the reserved metadata label 6862 (hexadecimal 0x1ace), as outlined in [CIP-0017](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0017) preferably without any of the optional fields.
+
+Example:
+
+``` Json
+{
+  "pools": [
+    {
+      "ticker": "DARK",
```

Comment:
I believe that putting stake keys here instead of pool ids/tickers would be better, because:

- there wouldn't be a need to update the metadata if delegation changes, but not the weight
- wallet implementations will have to perform one series of lookups less: instead of caring about pools, they will be only concerned with stake keys
- removing a delegation in favor of another would be as easy as re-delegating stake for one stake key - again, no metadata update

Reply:
As a separate note, there is probably no need in supporting tickers at all: reverse lookups only introduce complexity for the implementors to deal with

Reply:
dApps who will actually make use of this standard may not care about delegations at all. So why force them to perform additional lookups to figure out which of the stake addresses correspond to which pools?

Reply:
thanks @klntsky I was only referring to your 2nd https://github.com/cardano-foundation/CIPs/pull/628#discussion_r1412220561 in my own https://github.com/cardano-foundation/CIPs/pull/628#discussion_r1412254242; apologies for the ambiguity & I've tried to correct that now.  BTW I would support the suggestion to use stake addresses vs. pool IDs.

Reply:
> The users also interact with the delegation from the point of view of pools not stake keys, as in, users select pools and percentages and this information would need to be converted to key hashes in order to create the metadata as you are suggesting

You are only considering wallet's and user's point of view. But what about dApps that just want to generate change such that the distribution is preserved. These dApps may as well be other wallets that do not fully support this feature (i.e. in the UI), but try to respect the distribution.

I think it's obvious that one can always go from poolIDs to stake addresses and in the opposite direction. Stake addresses just have strictly more use cases without an extra lookup